### PR TITLE
Leksah

### PR DESCRIPTION
**Pull request by tomesh**
Add installation guide for Leksah, the Haskell IDE.

### CODE REVIEW

File: tech/languages/haskell/leksah.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+title: Leksah
+subsection: haskell
+order: 4
+---
+
+# Leksah
+
+[Leksah](http://leksah.org/) is a Haskell IDE written in Haskell using Gtk. It offers Workspaces for complex project with multiple
```
Comment (phracek) [inline tech/languages/haskell/leksah.md]: I think "Workspaces" should be either bolded or lowercased. Can you please rephrase it a bit?

File: tech/languages/haskell/leksah.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+subsection: haskell
+order: 4
+---
+
+# Leksah
+
+[Leksah](http://leksah.org/) is a Haskell IDE written in Haskell using Gtk. It offers Workspaces for complex project with multiple 
+packages with automatic build of dependencies. It enables standard IDE features (debugging, auto completition etc.).
+
+To install Leksah you need to install `ghc` and `cabal` first (or install the Haskell Platform).
```
Comment (phracek) [inline tech/languages/haskell/leksah.md]: What about to write a sentence like:

To install Leksah run command:

```
sudo dnf install ghc cabal
```
 or install Haskell Platform by command:

```
sudo dnf install ....
```

**Comment by tomesh** [comment]
I have made the requested update. Now it should be ok :)

**Comment by asamalik** [comment]
Thanks!

**Pull request merged**